repository/l.json

```diff
@@ -1717,6 +1717,19 @@
 				}
 			]
 		},
+		{
+			"name": "LSP-luau",
+			"details": "https://github.com/alihsaas/LSP-luau",
+			"labels": [
+				"lsp", "roblox", "luau"
+			],
+			"releases": [
+				{
+					"sublime_text": ">=3092",
```

Comment:
Your package is only compatible with ST4 since you are using LSP API that is only available there.

Reply:
This uses the server provided in https://github.com/NightrainsRbx/RobloxLsp which adds support for Roblox's Luau